src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -436,7 +436,7 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
             </View>
           </GenericSectionItem>
         </Section>
-        {inspectableTokenWarningText && (
+        {inspectableTokenWarningText && !phoneNumber && (
```

Comment:
Having `phoneNumber` means that "this ticket is sent to `phoneNumber`, then we should hide the warning.